src/geoserverx/cli/geofence.py

```
@@ -0,0 +1,41 @@
+import typer
+from rich import print
+
+from geoserverx._sync.gsx import SyncGeoServerX
+
+geofence_app = typer.Typer()
+
+
+@SyncGeoServerX.exception_handler
```

In the spirit of #83 we don't really want to have an exception handler coming from core that is building nice error objects.

According to our previous discussion, the architecture to pursue here is one where:

- If there is an error in core, then an exception is raised
- Calling code (in this case this CLI client) is responsible for catching the exception and dealing with it

With this in mind, let's not have an exception handler which is actually a method of a class in core.

To be more blunt, get rid of `@SyncGeoServerX.exception_handler`.

For now you can ignore exception handling. When you tackle #83 we can think about how to catch the exceptions here in the CLI